lib/Dialect/Arc/Transforms/GroupResetsAndEnables.cpp

```diff
@@ -143,6 +205,7 @@ LogicalResult GroupResetsAndEnablesPass::runOnModel(ModelOp modelOp) {
   RewritePatternSet patterns(&context);
   patterns.insert<ResetGroupingPattern>(&context);
   patterns.insert<EnableGroupingPattern>(&context);
+  patterns.insert<GroupAssignmentsInIfPattern>(&context);
```

Comment:
Absolutely no need to change this, but just that you are aware that it is also possible:

```suggestion
  patterns.add<ResetGroupingPattern, EnableGroupingPattern, GroupAssignmentsInIfPattern>(&context);
```